### PR TITLE
AC-579: Added Privacy Policy link to Settings page

### PR DESCRIPTION
**Pull request by deepak140596**
## Description of what I changed

Added link to Privacy Policy to the Settings Page of the app.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-579

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by deepak140596** [comment]
@VANKINEENITAWRUN, @f4ww4z  , this is the new PR.
Also I found a bug. Will update it on Talk.

**Comment by deepak140596** [comment]
AppVeyor build has failed. Can someone figure out what went wrong?

**Comment by Abdelaty** [comment]
Hey @deepak140596 thank you for your PR. 
About the AppVeyor build failed, you can check the console message from [here](https://ci.appveyor.com/project/AvijitGhosh82/openmrs-contrib-android-client/builds/23997601)
And one important thing, you didn't added tests, so you should unchecked this option.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsRecyclerViewAdapter.java [hunk 1]
```diff
@@ -87,7 +94,7 @@ public int getItemCount() {
         return mItems.size();
     }
 
-    class SettingsViewHolder extends RecyclerView.ViewHolder{
+    class SettingsViewHolder extends RecyclerView.ViewHolder {
```
Comment (Abdelaty) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsRecyclerViewAdapter.java]: Always don't forget to reformat the code before commit "D

**Pull request merged**